Gemfile.lock

```diff
@@ -138,13 +172,14 @@ PLATFORMS
 
 DEPENDENCIES
   aws-sdk-s3 (~> 1)
+  guard-rspec
   mysql2
   puma
   rack-test
   rake (~> 13.0)
   require_all
   rspec
-  rubocop-govuk (~> 3.16.0)
+  rubocop-govuk
```

Comment:
This one's my fault.  The version number should not have been pinned in the first place.  It's a mistake I introduced in the master branch.  Bundler has spotted it and removed the redundant pin; we might as well let the correction be made in this PR.